Assignment 2.Rmd

```diff
@@ -87,14 +93,70 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+``` {r}
+library(tidyr)
+library(dplyr)
+
+STU.SUM <- EDGE%>%group_by(to)%>%summarise(sum(count)) 
+TE.SUM <- EDGE%>%group_by(from)%>%summarise(sum(count)) 
+names(STU.SUM) <- c("id","count")
+names(TE.SUM) <- c("id","count")
+TOTAL <- bind_rows(STU.SUM,TE.SUM)
+VERTEX2 <- full_join(VERTEX,TOTAL,by="id")
 
+library(igraph)
+
+g2 <- graph.data.frame(EDGE, directed=TRUE, vertices=VERTEX2)
+plot(g2,layout=layout.fruchterman.reingold,vertex.color=VERTEX2$gender,vertex.size=VERTEX2$count)
+```
```

Comment:
Great graph! You may want to size the vertices using `log()` so that the size between the teacher and student vertices are less extreme.